content/gaad-2021.md

```diff
@@ -0,0 +1,39 @@
+
+## What We've Done So Far
+
+Ember has also been shipping improvements with the “accessibility by default” developer experience in mind. These improvements include support for default app language (RFC [#635](https://emberjs.github.io/rfcs/0635-ember-new-lang.html)), default page titles (RFC [#645](https://emberjs.github.io/rfcs/0645-add-ember-page-title-addon.html)), added [content to the guides](https://guides.emberjs.com/release/accessibility/application-considerations/) specifically for accessibility, changes to the [website design](https://ember-styleguide.netlify.app/concepts/colors/) for better accessibility, and addition of accessibility-related linting rules to [Ember Template Lint](https://github.com/ember-template-lint/ember-template-lint). Members of the Accessibility Working Group also published two new addons, [ember-context-id-helper](https://github.com/KamiKillertO/ember-context-id-helper) and [ember-select-light](https://github.com/ember-a11y/ember-select-light), to give developers more tools to craft accessible code.
+
+## What We Have Planned
+
+Looking forward, we have even more accessibility-related work planned. Two RFCs have been merged that target specific improvements; one addresses a way to make it easier to associate elements through unique id attributes in template-only components (RFC [#659](https://emberjs.github.io/rfcs/0659-unique-id-helper.html)), and the other proposes an interactive way to create new Ember apps, ensuring that accessible outcomes are front and center (RFC [#638](https://emberjs.github.io/rfcs/0638-interactive-app-creation.html)).
+
+We will continue on our journey of accessibility with more improvements to the guide prose and code samples, as well as implement targeted improvements to the design of the guides and API documentation. Our goal is to reach [WCAG’s AA level of conformance on our official project websites](https://ember-styleguide.netlify.app/concepts/accessibility/), and we are determined to see this through to completion.
+
+## How You Can Help
+
+We also invite everyone to help make Ember a more accessible framework. There are lots of ways to get involved! Here are a few ideas:
+
+- Help maintain a11y-specific addons in the [Ember A11y](https://github.com/ember-a11y) organization.
+- Contribute [additional template linting rules](https://github.com/ember-template-lint/ember-template-lint/issues?q=is%3Aopen+is%3Aissue+label%3Aa11y) to Ember Template Lint
+- Tackle an accessibility-related [Help Wanted issue](https://help-wanted.emberjs.com/ember-a11y)
```

Comment:
suggestion (non-blocking): The issues here are ~2yrs old, any a11y issues we could add as `help wanted` here?

Reply:
great suggestion. I'll look in the relevant repositories too see if there are issues that would be appropriate to label.